internal/pkg/billing/billing_utils_test.go

```diff
@@ -0,0 +1,22 @@
+package billing
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFormatPrice(t *testing.T) {
+	assert.Equal(t, "$1.00 USD/GB", FormatPrice(1, "GB"))
+	assert.Equal(t, "$1.20 USD/GB", FormatPrice(1.2, "GB"))
+	assert.Equal(t, "$1.23 USD/GB", FormatPrice(1.23, "GB"))
+	assert.Equal(t, "$1.23456 USD/GB", FormatPrice(1.23456, "GB"))
```

Comment:
We really only need one of these assertions, since these cases are covered in the test underneath now!